x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/min_age_input.js

```diff
@@ -4,44 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
-import { LearnMoreLink } from '../../components';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
 import {
   EuiFieldNumber,
   EuiFlexGroup,
   EuiFlexItem,
   EuiFormRow,
   EuiSelect,
 } from '@elastic/eui';
+
 import {
   PHASE_ROLLOVER_MINIMUM_AGE,
   PHASE_ROLLOVER_MINIMUM_AGE_UNITS,
 } from '../../../constants';
+import { LearnMoreLink } from '../../components';
 import { ErrableFormRow } from '../form_errors';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
-const MinAgeInputUi = props => {
-  const { rolloverEnabled, errors, phaseData, phase, setPhaseData, isShowingErrors, intl } = props;
+
+export const MinAgeInput = props => {
```

Comment:
TODO: Confirm this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351801-12549800-8cd4-11e9-9d35-3696e9c96906.png)